src/views/Batches/batchComponents/BatchForm.jsx

```diff
@@ -489,7 +495,19 @@ const BatchForm = (props) => {
                       placeholder=""
                     />
 
-                </div>
+                  </div>
+                  <div className="col-md-6 col-sm-12 mt-2">
+                       <Input
+                        control="lookup"
+                        icon="down"
+                        name="mode_of_payment"
+                        label="Mode of Payment"
+                        required
+                        placeholder="Mode of Payment"
+                        className="form-control"
+                        options={paymentOptions}
```

Comment:
is this a required field or optional field? If optional, what would be the default value for this?

Reply:
what will be the default value for this field.